src/Domain.Tests/GameManagerTests.cs

```diff
@@ -79,126 +95,50 @@ public async Task CreateGame_SavesGameInGameRepository()
         }
 
         [Fact]
-        public async Task Move_FirstMove_CorrectMoveNumber()
```

Comment:
Why was this removed?

Reply:
Figuring out the move number is no longer part of the GameManager